### DISCUSSION TITLE
Calibration support

### DISCUSSION THREAD

**Discussion by phil-barrett** [terjeio/ioSender#122] [Ideas]
Mach 4 (3 also?) has calibration support.  You tell it to move an axis a certain distance and then enter how far it actually moved. It then computes the actual steps/unit distance and sets it.

I think this would be a very useful addition to ioSender, particularly for newbies.  Should work with any Grbl version.

**Comment by terjeio** [comment]
Yep - that would be useful. More work to put on the todo list...
